great_expectations/dataset/util.py

```diff
@@ -297,3 +297,253 @@ def continuous_partition_data(data, bins='auto', n_bins=10):
         "bins": bin_edges,
         "weights": hist / len(data)
     }
+
+
+def infer_distribution_parameters(data, distribution, params=None):
+    """Convenience method for determining the shape parameters of a given distribution
+
+    Args:
+        data (list-like): The data to build shape parameters out of
+        distribution (string): Scipy distribution, determines which shape parameters to return
+        params (dict or None): The known parameters of desired distribution (`data`)
+                       Keep as None to infer necessary parameters from the data data
+
+    Returns:
+        A dictionary of named parameters::
+
+        {
+            "mean": (float),
+            "std_dev": (float),
+            "loc": (float),
+            "scale": (float),
+            "alpha": (float),
+            "beta": (float),
+            "min": (float),
+            "max": (float),
+            "df": (float)
+        }
+
+        See: https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.kstest.html#scipy.stats.kstest
+    """
+
+    if params is None:
```

Comment:
I think this is (perhaps unavoidably because we inherit the confusion from scipy) confusing, but I say let's go with it for now.

Reply:
Am totally up for any suggestions. I'll commit some changes that may clear it up a bit more.